<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>t_page</class>
 <widget class="QWidget" name="t_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>选择1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>选择2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>选择3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>选择4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>选择5</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <row>
      <property name="text">
       <string>1-2节</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3-4节</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5-6节</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7-8节</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9-10节</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>周一</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周二</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周三</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周四</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周五</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周六</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周日</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_del">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>删除该表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_del_all">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>删除全部表</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>tableWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
